### ISSUE TITLE
[Ente Auth] Themed icon (Android) does not adhere to requirements

### ISSUE THREAD

**Issue by Wikiwix** [ente-io/ente#3840]
### Description

For Ente Auth the themed icon on Android has a shadow which seems a little out of place and violates [the spec](https://developer.android.com/develop/ui/views/launch/icon_design_adaptive#user-theming):
> - Use icons with clean edges. The layers must not have masks or background shadows around the outline of the icon.

A screenshot with some icons for reference:

![share_7681630843850031885](https://github.com/user-attachments/assets/26090807-6c47-492b-8f41-662dffd0bec1)

## Notes on implementing a fix

- I checked, and I can only find the pngs in the repo, which makes removing the shadow harder than it needs to be
- the `dev_dependency` `flutter_icons` is deprecated, inactive for 4 years, not Dart 3 compatible and does not explicitly support monochrome icons. [`flutter_launcher_icons`](https://pub.dev/packages/flutter_launcher_icons) appears to be a good alternative that explicitly supports monochrome icons

### Version

4.0.2

### What product are you using?

Ente Auth

### What platform are you using?

Mobile - Android

**Comment by Bl4ckspell7** [comment]
Yes, ever since I enabled monochrome icon in https://github.com/ente-io/ente/pull/2118, the shadow annoys me. I finally found the svg source files ([figma](https://www.figma.com/design/SYtMyLBs5SAOkTbfMMzhqt/Ente-Visual-Design)). I am working on a pull request.
